src/ext/oblivc/obliv_common.h

```diff
@@ -86,6 +87,7 @@ void honestOTExtRecv1Of2Chunk(void* vargs,char* dest,int nchunk,
     int len,bool isCorr);
 
 struct HonestOTExtSender* honestOTExtSenderNew(ProtocolDesc* pd,int destparty);
+struct HonestOTExtSender* honestOTExtSenderSplit(struct HonestOTExtSender* t, ProtocolDesc* pd);
```

Comment:
SenderNew in the line above uses (pd, dest), SenderSplit uses (sender_stuff, pd). How about reordering arguments here? And let's start adding comments on function headers, single line comments are enough for now.
Finally, I think the rest of the file uses `s` for sender, and `r` for receiver. `t` is kind of awkward here. How about `s_src` or `s_parent` or something like that?

I know I don't set a great example wrt API consistency, but let's try to improve on that:)